test/e2e/commands/selectProcessor.js

```diff
@@ -0,0 +1,26 @@
+exports.command = function(selector, callback) {
```

Comment:
Do you even need this file now when you have  selectCssProcessor and selectJsProcessor?